openmls/src/ciphersuite/signable.rs

```diff
@@ -54,12 +54,16 @@ pub trait Signable: Sized {
         self,
         backend: &impl OpenMlsCryptoProvider,
         credential_bundle: &CredentialBundle,
-    ) -> Result<Self::SignedOutput, CredentialError>
+    ) -> Result<Self::SignedOutput, CredentialBundleError>
     where
         Self::SignedOutput: SignedStruct<Self>,
     {
-        let payload = self.unsigned_payload()?;
-        let signature = credential_bundle.sign(backend, &payload)?;
+        let payload = self
+            .unsigned_payload()
+            .map_err(|_| CredentialBundleError::LibraryError)?;
```

Comment:
Do we want to keep the practice of annotating LibraryErrors with a reason on why they shouldn't occur in a particular situation? I think it would help code readability quite a bit.

Reply:
I see. Then I would propose that until we have that error, we use code comments to annotate the error, so that we can easily populate the error string later.

In this particular case: What is hiding behind the "unlikely"? If there is a chance this errors out, it probably shouldn't be a library error.